benchmarks/Cargo.toml

```
@@ -11,7 +11,7 @@ milli = { path = "../milli" }
 jemallocator = "0.3.2"
 
 [dev-dependencies]
-heed = "*" # we want to use the version milli uses
+heed = { git = "https://github.com/Kerollmops/heed", tag = "v0.12.0", default-features = false, features = ["lmdb", "sync-read-txn"] }
```

```suggestion
heed = { git = "https://github.com/Kerollmops/heed", tag = "v0.12.0" }
```

It is not mandatory to specify the features, IIRC.

---

That's right thanks :grinning: